WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsUtils.java

```diff
@@ -127,7 +130,8 @@ public static void refreshMetadataNewUser(String username, String email) {
         metadata.setNumBlogs(1);
         metadata.setUsername(username);
         metadata.setEmail(email);
-        metadata.setGutenbergEnabled(AppPrefs.isGutenbergDefaultForNewPosts());
+        // GB is enabled for new users
+        metadata.setGutenbergEnabled(true);
```

Comment:
Shouldn't this be reflected somehow in the implementation of [SiteUtils.isBlockEditorDefaultForNewPost](https://github.com/wordpress-mobile/WordPress-Android/blob/3346b32c858c26f5200aa2a87895c5f1f259e8aa/WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java#L21) too?

Reply:
This code is used when a new account is created. It's "kind" of safe to set it to `true` here, since we've enabled the block editor on new users.
We can although change and use the `SiteUtils.isBlockEditorDefaultForNewPost ` method call, but change may not be super trivial since a call comes from the LoginLibrary. 
Need to check this more deeper though.